### ISSUE TITLE
Create "heart-beat" alert to verify that the github-receiver is working

### ISSUE THREAD

**Issue by stephen-soltesz** [m-lab/prometheus-support#149]
On new releases of the github-receiver it should be possible to quickly verify that it is working correctly without waiting until the next alert fires.

A proposal: 

Add static firing alerts based on recording rules. e.g. 

```
heartbeat:vector = vector(1)
```

Define an alert when that value is high.

```
AlertmanagerHeartbeatAlive
IF heartbeat:vector > 0
```

This should result in an issue opening. That issue should remain open indefinitely. When new alertmanager or github-receiver are released, we would close the issue and wait to see that a new one was recreated, verifying that the AM and GR work e2e.

**Comment by nkinkade** [comment]
I'm don't love the idea of having a Github issue perpetually open, and an alert perpetually firing. No automated solution I can think of, so far, appeal to me much.

One idea is to configure a testing alert with a hair trigger (true for >1m) that will only fire under very specific conditions that we orchestrate. For example, in prometheus-support, Travis-CI has SSH access to EB, and on EB there is a node_manager instance running. Perhaps the Travis-CI build could create a file like EB:/var/spool/nodeexporter/test-alerting.prom, with content that would trigger the testing alert very quickly, then the build could remove that file once enough time has elapsed that the alert should have triggered.

**Comment by stephen-soltesz** [comment]
I agree that having an issue perpetually open is not great. I'm wondering is there a way to leverage the GMX to *cause* an alert but normally that alert would be quiet.

For example, to test alerts we could:

* gmx -> some_fake_metric
* prometheus scrapes GMX
* alert for count(some_fake_metric) > 0 fires
* AM receives alert
* AM sends a message to slack (in staging and prod) and github receiver (in prod)

This would be a manual step, but it would be available anytime we wanted to be sure things were all working together.

Or, could there be an always-firing alert that we *don't* route to slack or github issues but (somehow) also monitor for the sending and receiving of at the AM? maybe some /dev/null like thing with a notification counter?
